src/routes/local.ts

```diff
@@ -26,7 +27,9 @@ router.get('/satellite', (req, res) => {
         return SatelliteUtilities.getSatelliteByName(req.query.name)
             .then((satelliteDoc) => {
                 if (!satelliteDoc) return res.sendStatus(400)
-                return res.json(_.pick(satelliteDoc, PROPS_TO_SEND_IN_RESPOSNE))
+                const satellitesArray = <Partial<SatelliteInterface>[]> []
```

Comment:
You can use Pick<SatelliteInterface, "officialName" | "_id" ...> instead of Partial<>